scripts/style.sh

```diff
@@ -7,6 +7,7 @@ find . \
     -name 'FirebaseCommunity.h' -prune -o \
     -name 'Messaging' -prune -o \
     -name 'Storage' -prune -o \
+    -name 'Pods' -prune -o \
     -name '*.[mh]' \
     -not -name '*.pbobjc.*' \
-    -exec clang-format -style=file -i '{}' \;
+    -print0 | xargs -0 clang-format -style=file -i \
```

Comment:
This trailing backslash is not required (and confusing).

The \; in the find -exec version is to tell the exec program the end of the invocation (while escaping the semicolon from interpretation by bash). No such end of statement is required here.